src/public/modules/core/components/toolset-dropdown.client.component.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+angular.module('core').component('toolsetDropdownComponent', {
+  templateUrl: 'modules/core/componentViews/toolset-dropdown.html',
+  controller: ['$scope', toolsetDropdownController],
+  controllerAs: 'vm',
+})
+
+function toolsetDropdownController($scope) {
+  /**
+   * Logic for dropdown component. Add to labelAndLink to add more websites to the dropdown.
+   */
+  const vm = this
+
+  vm.isDropdownHover = false
+  vm.isDropdownFocused = false
+  vm.isDropdownOpen = false
+  vm.labelAndLink = [
```

Comment:
Should we add a link to [parkingsg](https://www.parking.sg/) as well?

Reply:
Answer from PM @mantariksh: The links here are targeted for public officers who may not be aware of other tools that could be useful for their workflows (just like FormSG is). parkingsg doesn't fit that purpose since it's for citizens, not for officers.